src/main/java/frc/robot/subsystems/NeoMotorDriveSystem.java

```diff
@@ -81,18 +82,29 @@ public NeoMotorDriveSystem()
     
   }
 
+  public double deadBand(double inp)
```

Comment:
This function should have a more descriptive name. Quickly thinking, maybe something like "enforceDeadBand". The current name makes it sounds like it's "deadBanding" something.